blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/Containers.java

```diff
@@ -39,7 +38,7 @@ public T get(String name) {
             @Override
             public Iterator<T> iterator(int start, int limit) {
                 if (start >= base.size()) {
-                    return Collections.<T>emptyList().iterator();
+                    return Collections.emptyIterator();
```

Comment:
Is this required? If you want to do general code cleanup that should be another PR.

Reply:
well while I'm doing changes (i.e almost browsing all classes) I applied very easy cleanup (obvious simplify code)